File: Fuentes/Principal/Logica/DatosArea.cs
using System;
using System.Collections.Generic;
using System.Data.SqlClient;
using System.Linq;
using System.Text;

namespace Logica
{
    public class DatosArea
    {
         
        private int id;
        private string nombre; 
        private string clave;

        public int Id
        {
            get { return this.id; }
            set { this.id = value; }
        }
        public string Nombre
        {
            get { return this.nombre; }
            set { this.nombre = value; }
        }
        public string Clave
        {
            get { return this.clave; }
            set { this.clave = value; }
        }

        public void ObtenerParametrosInformacionArea()
        {

            string[] parametros = Environment.GetCommandLineArgs().ToArray();
            if ((parametros.Length > 1)) {
	            int numeracion = 26;
	            this.Id = Convert.ToInt32(parametros[numeracion].Replace("|", " ")); numeracion += 1;
                this.Nombre = parametros[numeracion].Replace("|", " "); numeracion += 1;
                this.Clave = parametros[numeracion].Replace("|", " "); numeracion += 1; 
            }

        }
    }
}

File: Fuentes/Actividades/Logica/DatosArea.vb
Public Class DatosArea

    Private id As Integer
    Private nombre As String
    Private clave As String

    Public Property EId() As Integer
        Get
            Return Me.id
        End Get
        Set(value As Integer)
            Me.id = value
        End Set
    End Property
    Public Property ENombre() As String
        Get
            Return Me.nombre
        End Get
        Set(value As String)
            Me.nombre = value
        End Set
    End Property
    Public Property EClave() As String
        Get
            Return Me.clave
        End Get
        Set(value As String)
            Me.clave = value
        End Set
    End Property

    Public Sub ObtenerParametrosInformacionArea()

        Dim parametros() = Environment.GetCommandLineArgs().ToArray()
        If (parametros.Length > 1) Then
            Dim numeracion As Integer = 26
            Me.EId = Convert.ToInt32(parametros(numeracion).Replace("|", " ")) : numeracion += 1
            Me.ENombre = parametros(numeracion).Replace("|", " ") : numeracion += 1
            Me.EClave = parametros(numeracion).Replace("|", " ") : numeracion += 1
        End If

    End Sub

End Class
